manager/src/main/java/org/openremote/manager/dashboard/DashboardStorageService.java

```diff
@@ -130,10 +134,9 @@ protected <T extends Dashboard> T update(T dashboard, String userId) {
         return persistenceService.doReturningTransaction(em -> {
             Dashboard d = em.find(Dashboard.class, dashboard.getId());
             if(d != null) {
-                if(d.getEditAccess() == DashboardAccess.PRIVATE) {
-                    if(!(d.getOwnerId().equals(userId))) {
+                if(d.getEditAccess() == DashboardAccess.PRIVATE && !(d.getOwnerId().equals(userId))) {
                         throw new WebApplicationException("You are not allowed to edit this dashboard!", FORBIDDEN);
```

Comment:
This is technically leaky as WebApplicationException is a JAXRS exception but this code isn't a JAXRS class